mysql/resource_rds_config.go

```diff
@@ -0,0 +1,182 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceRDSConfig() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: CreateRDSConfig,
+		UpdateContext: UpdateRDSConfig,
+		ReadContext:   ReadRDSConfig,
+		DeleteContext: DeleteRDSConfig,
+		Importer: &schema.ResourceImporter{
+			StateContext: ImportRDSConfig,
+		},
+		Schema: map[string]*schema.Schema{
+			"binlog_retention_period": {
```

Comment:
Would you consider adding "hours" to name somewhere? Maybe `binlog_retention_hours`.

If your naming corresponded to the real names somewhere, then I'd be more ok with that. Still, if you can write documentation in naming, please do it - and people won't have to read the documentation to understand, what that thing means.

Reply:
Yes, much better. Now the config parameters are the same as in AWS docs.